lib/lob/resource_base.ex

```diff
@@ -18,27 +18,59 @@ defmodule Lob.ResourceBase do
         def list(params \\ %{}, headers \\ %{}) do
           Client.get_request("#{base_url()}?#{Util.build_query_string(params)}" , Util.build_headers(headers))
         end
+
+        @spec list!(map, map) :: {map, list}
+        def list!(params \\ %{}, headers \\ %{}) do
+          case list(params, headers) do
+            {:ok, body, headers} -> {body, headers}
+            error -> raise to_string(error)
```

Comment:
Don't you want to raise the error message not the whole tuple? Something like this?

```elixir
{:error, error} -> raise(error)
```